ci/tasks/destroy-infrastructure.sh

```diff
@@ -13,12 +13,13 @@ popd > /dev/null
 trap save_terraform_state EXIT
 
 pushd "${release_dir}/src/omg-tf"
+        configure_terraform_backend
 	terraform init
 	terraform get
-	yes "yes" | terraform destroy --parallelism=100 -state=${terraform_state} -var-file=${terraform_config} && exit 0
+	yes "yes" | terraform destroy --parallelism=100 -var-file=${terraform_config} && exit 0
```

Comment:
I think ou can use the `-input=false` flag to avoid the yes pipe. Terraform has a good guide about how to [run it in automation](https://www.terraform.io/guides/running-terraform-in-automation.html).